packages/gatsby-theme-store/src/components/Minicart/Drawer/Content.tsx

```diff
@@ -1,65 +1,56 @@
 import React from 'react'
-import { Flex, Text } from 'theme-ui'
+import { Flex, Box } from 'theme-ui'
 import type { FC } from 'react'
 import { useIntl } from '@vtex/gatsby-plugin-i18n'
 
 import { useNumberFormat } from '../../../sdk/localization/useNumberFormat'
-import MinicartDelete from './Delete'
-import { HeaderMinicartDrawerContentImage } from './Image'
-import MinicartQuantity from './Quantity'
-import type { OrderForm } from '../../../sdk/orderForm/Provider'
-import { IMAGE_DEFAULT } from '../../../sdk/product/constants'
+import type { OrderFormData } from '../../../sdk/product/useAvailability'
+import { useAvailability } from '../../../sdk/product/useAvailability'
+import { UnavailableItems } from './ContentItems/UnavailableItems'
+import { AvailableItems } from './ContentItems/AvailableItems'
 
 export interface MinicartContentProps {
-  data: NonNullable<OrderForm>['items']
+  data: OrderFormData
```

Comment:
What do you think about inline it `OrderForm['items']` or change the name of this type, maybe or something like this `OrderFormItems`?